.github/ISSUE_TEMPLATE/new_webpage.yml

```diff
@@ -0,0 +1,112 @@
+name: ✨ New Webpage
+description: Start here to request a new page
+title: "✨ {{ TITLE }} "
```

Comment:
Should we use the page emoji instead? 📄

Sparkle is meant to be used for features (code)